Kickstarter-iOS/Views/Controllers/SearchViewController.swift

```diff
@@ -124,6 +125,13 @@ internal final class SearchViewController: UITableViewController {
       .observeValues { [weak self] in
         self?.changeSearchFieldFocus(focus: $0, animated: $1)
     }
+
+    self.viewModel.outputs.scrollToProjectRow
+      .observeForControllerAction() // NB: this also is required to prevent NSException.
```

Comment:
noticed this was crashing with `observeForUI`. Something special about the Search controller?